### PR TITLE
Release notes for v1.2

### PR DESCRIPTION
**Pull request by registerrier**
**Description**

This pull request prepares the addition of the release notes for v1.2.

For now it is a complete draft. Many things are still missing 
Please suggest missing points, propose formulation etc.

**Dear reviewer**

### CODE REVIEW

File: docs/release-notes/v1.2.rst [hunk 1]
```diff
@@ -1,6 +1,206 @@
...
+- Correct ``MapDataset.info_dict()`` to use background model rather than IRF background when
+  giving excess counts and significance.
+- Import ray only when needed.
+- Added information on number of degrees of freedom on ``FluxMaps`` and ``FluxPoints`` objects.
+- Reduced memory usage of ``MapEvaluator`` and ``PSFMap.get_psf_kernel()``.
+- Added support for multiprocessing in ``FluxProfileEstimator``.
+- Added multiprocessing for ``WcsNDMap`` convolution.
+- Add a context manager for multiprocessing.
+- Add a faster reprojection method : ``reproject_by_image``.
+- Use interpolation for dardk matter mass. Add Zhao profile and
```
Comment (Astro-Kirsty) [inline docs/release-notes/v1.2.rst:83]:
```suggestion
- Use interpolation for dark matter mass. Add Zhao profile and
```

File: docs/release-notes/v1.2.rst [hunk 2]
```diff
@@ -1,6 +1,206 @@
...
+- Update 4FGL to DR4.
+- Added 1LHAASO catalog.
+
+*gammapy.data*
+
+- A general scheme for metadata support has been introduced. The ``Metadata`` base
+  class has been designed according to PIG X.
+- Added a function to remove a time interval from a ``GTI``.
+- Added function to export part of a ``DataStore`` to an IVAO compliant ObsCore table.
+
```
Comment (Astro-Kirsty) [inline docs/release-notes/v1.2.rst:34]: There was the "ObservationsEventsSampler" class

File: docs/release-notes/v1.2.rst [hunk 3]
```diff
@@ -1,6 +1,206 @@
...
+*gammapy.makers*
+
+- Prototype support for asymmetric IRF in Gammapy's ``Maker`` classes.
+  A tutorial exposing how to create such IRFs has been added.
+
```
Comment (Astro-Kirsty) [inline docs/release-notes/v1.2.rst:39]: The `SafeMaskMaker` at DL3 level

File: docs/release-notes/v1.2.rst [hunk 4]
```diff
@@ -1,6 +1,206 @@
...
+- Add function to determine pivot energy for all spectral models.
+- Added position as a parameter for the ``TemplateSpatialModel``. Add a caveat.
+
+*gammapy.estimators*
+
+- Add a ``slice_by_coord()`` function on ``FluxMaps``.
+- Introduce timing utility functions: point to point flux variance, fractional excess
+  variance
+- Add optional sensitivity estimation in ``ExcessMapEstimator``.
+
```
Comment (Astro-Kirsty) [inline docs/release-notes/v1.2.rst:56]: The `EnergyDependentMorphologyEstimator`

File: docs/release-notes/v1.2.rst [hunk 5]
```diff
@@ -1,6 +1,206 @@
...
+Summary
+~~~~~~~
+
+New features
+~~~~~~~~~~~~
+
+- New helper classes have been added to perform event sampling.
+- A checksum option has been added on read and write methods in Gammapy. It follows the
+  FITS standard and reuses the astropy methods and behavior. A checksum for yaml file
```
Comment (Astro-Kirsty) [inline docs/release-notes/v1.2.rst:19]:
```suggestion
  FITS standard and reuses the astropy methods and behaviour. A checksum for yaml file
```

**Review by Astro-Kirsty — COMMENTED** [review]
Looks good so far, thanks for the effort @registerrier !

File: docs/release-notes/v1.2.rst [hunk 6]
```diff
@@ -1,6 +1,206 @@
...
+
+*gammapy.maps*
+
+- Implement ``TimeMapAxis.pix_to_coord()``
+
+*gammapy.modeling*
+
+- Added ``Prior`` class.
+- Add function to determine pivot energy for all spectral models.
+- Added position as a parameter for the ``TemplateSpatialModel``. Add a caveat.
```
Comment (AtreyeeS) [inline docs/release-notes/v1.2.rst:48]:
```suggestion
- Added position as a parameter for the ``TemplateSpatialModel``. CAVEAT: results are correct only when the fitted position is close to the map center.
- Added Spatial parameters in FoVBackgroundModels
- SkyModel evaluation now supports a TimeMapAxis
- Adapt FluxPointsDataset to directly fit lightcurves
```

File: docs/release-notes/v1.2.rst [hunk 7]
```diff
@@ -1,6 +1,206 @@
...
+- Added function to export part of a ``DataStore`` to an IVAO compliant ObsCore table.
+
+*gammapy.makers*
+
+- Prototype support for asymmetric IRF in Gammapy's ``Maker`` classes.
+  A tutorial exposing how to create such IRFs has been added.
+
+*gammapy.maps*
+
+- Implement ``TimeMapAxis.pix_to_coord()``
```
Comment (AtreyeeS) [inline docs/release-notes/v1.2.rst:42]:
```suggestion
- Implement ``TimeMapAxis.pix_to_coord()``
- Implement ``TimeMapAxis.to_gti()``
```

**Review by AtreyeeS — APPROVED** [review]
Thanks @registerrier ! Added some more details for the new features

File: docs/release-notes/v1.2.rst [hunk 8]
```diff
@@ -1,6 +1,206 @@
...
+- [#4835] Management of metadata for `Models` (Bruno Khélifi)
+- [#4834] Adding prior stat sum to datasets (Katrin Streil)
+- [#4829] Caching gti and radmax (REGEARD Maxime)
+- [#4828] Adapt SkyModel to evaluate on TimeMapAxis (Atreyee Sinha)
+- [#4822] Add a function to delete a time interval from GTI (Claudio Galelli)
+- [#4817] Computation of total observation time map (Atreyee Sinha)
+- [#4814] Introduce a function to compute the doubling/halving time for a lightcurve (Claudio Galelli)
+- [#4810] Adding a tutorial for observational clustering (Astro-Kirsty)
+- [#4808] adding `Observations` in memory generator (REGEARD Maxime)
+- [#4805] Description of the arguments of the class `Observation' (Bruno Khélifi)
```
Comment (registerrier) [inline docs/release-notes/v1.2.rst:163]:
```suggestion
- [#4805] Description of the arguments of the class `Observation` (Bruno Khélifi)
```

**Comment by registerrier** [comment]
Merging this now, even if preliminary state, in order to perform feature freeze and v1.2.x branching

**Pull request merged**